patches/tModLoader/Terraria/Netplay.cs.patch

```diff
@@ -84,12 +84,15 @@
  							NetMessage.SendData(8, -1, -1, null, Main.player[Main.myPlayer].SpawnX, Main.player[Main.myPlayer].SpawnY);
  						}
  
-@@ -512,8 +_,6 @@
+@@ -512,8 +_,9 @@
  								Main.statusText = Connection.StatusText + ": " + (int)((float)Connection.StatusCount / (float)Connection.StatusMax * 100f) + "%";
  							}
  						}
 -
 -						Thread.Sleep(1);
++						/*instead of letting the thread that checks for new data continually run on the CPU 
++						// (thus taking away time from other processes) make the thread sleep for 1 millisecond between calls. */
++						Thread.Sleep(1); 
  					}
```

Comment:
This is present in vanilla, so just delete this entire specific patch instead?

Reply:
Is this fix confirmed? Like, it does make some sense but it's not on the main thread right?

Reply:
For Mirsario - there are other patches in Netplay.cs.patch that we're added and are independent to this, so I'm hesitant to delete the whole patch file. That said, I agree cleanup the patch file as seeing remove and add looks really ugly, just not sure how without directly editing it.

For Chicken-Bones - steam (social) connections invoke a new thread in class method StartSocialClient(). Likewise TCP connections invoke a new thread in StartTcpClient(). 
And just to confirm the fix via recordings in case any doubt.
Before: (on 1.4 branch)
https://user-images.githubusercontent.com/59670736/104056926-7d1acd80-51ae-11eb-804a-e89d22a8d9ed.mp4

 After: (on this branch)
https://user-images.githubusercontent.com/59670736/104057341-39749380-51af-11eb-98a7-607388173a3b.mp4

Reply:
I did not mention anything about files.

Reply:
Okay, then I'm lost on your request. 
Thread.Sleep(1) was removed 4+ years ago. I'm not sure why it's still showing up in this diff.
Thread.Sleep(1) + comment was added yesterday to the branch. That is the change.

Again, semi-new to patching, so I'm not sure why the removal statement is showing up. Realistically, the final patch file should just be add comment. I'll try and clean that up so its like that, but I'm only aware of direct edit the patch file to make that so

Reply:
I removed the patch myself. There's not much reason for the comment of what basically is just vanilla code.

On one of the videos you posted, you accidentally go into singleplayer.

Reply:
Oh, I see what you mean now. Yeah, makes sense. Thanks for making change. Main reason for comment is so that it didn't get removed later again by someone else, but yeah, vanilla is vanilla so shouldn't be touching anyways.

Yeah, in the base case I go into singleplayer for first 20ish seconds to show base CPU, then mutliplayer for remainder to show previous CPU. (hence why its 20 sec longer)
The new case I just do multiplayer since no point in doubling down on singleplayer recording (although I did open it in SP to be sure its consistent, separately).
Should have timestamped original post, fair point.

Reply:
I see, didn't notice that it's longer. I'll merge this now I guess.